WickedEngine/wiHelper.cpp

```diff
@@ -1410,8 +1415,12 @@ namespace wi::helper
 			WideCharToMultiByte(CP_UTF8, 0, from.c_str(), -1, &to[0], num, NULL, NULL);
 		}
 #else
-		std::wstring_convert<std::codecvt_utf8<wchar_t>> cv;
-		to = cv.to_bytes(from);
+		std::mbstate_t state = std::mbstate_t {};
+		const wchar_t* from_c = from.c_str();
+		const std::size_t len = 1 + std::wcsrtombs(nullptr, &from_c, 0, &state);
+		std::vector<char> to_c(len);
+		std::wcsrtombs(to_c.data(), &from_c, to_c.size(), &state);
```

Comment:
as discussed on discord, this fails if locale is not an UTF-8 locale, so we might need to use `setlocale`